.env.docker.scaleway.example

```
@@ -0,0 +1,172 @@
+RABBITMQ_RESULT_ROUTING_KEY=invoke-engine-result
+RABBITMQ_RESULT_QUEUE=virtual-contributor-invoke-engine-result
+RABBITMQ_INVOKE_ENGINE_EXPERT=virtual-contributor-engine-expert
+RABBITMQ_INVOKE_ENGINE_GUIDANCE=virtual-contributor-engine-guidance
+RABBITMQ_INVOKE_ENGINE_GENERIC=virtual-contributor-engine-generic
+RABBITMQ_INVOKE_ENGINE_OPENAI_ASSISTANT=virtual-contributor-engine-openai-assistant
+
+
+ALKEMIO_SERVER_ENDPOINT=http://host.docker.internal:4000/graphql
+KRATOS_API_PUBLIC_ENDPOINT=http://kratos:4433/
+SERVICE_ACCOUNT_USERNAME=notifications@alkem.io
+SERVICE_ACCOUNT_PASSWORD=change-me-now
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SERVICE_ACCOUNT_PASSWORD key should go before the SERVICE_ACCOUNT_USERNAME key

(UnorderedKey)